numpy/lib/shape_base.py

```diff
@@ -992,27 +988,25 @@ def dsplit(ary, indices_or_sections):
     array([[[  0.,   1.,   2.,   3.],
             [  4.,   5.,   6.,   7.]],
            [[  8.,   9.,  10.,  11.],
-            [ 12.,  13.,  14.,  15.]]])
+            [12.,  13.,  14.,  15.]]])
     >>> np.dsplit(x, 2)
-    [array([[[  0.,   1.],
-            [  4.,   5.]],
-           [[  8.,   9.],
-            [ 12.,  13.]]]),
-     array([[[  2.,   3.],
-            [  6.,   7.]],
-           [[ 10.,  11.],
-            [ 14.,  15.]]])]
+    [array([[[ 0.,  1.],
+            [ 4.,  5.]],
+           [[ 8.,  9.],
+            [12., 13.]]]), array([[[ 2.,  3.],
```

Comment:
Why did you need to rewrap this line but not the ones below?

Reply:
This is what NumPy produces on master branch locally for me -- as noted above this PR was initially aimed at getting doctest to pass and then relaxed to just require passing refguide, so the superset of changes in the PR probably exceeds the absolute minimum requirement for refguide to pass.

I'll mark this particular one as resolved since the change matches master branch output formatting.